CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# @envoyproxy/envoy-mobile-maintainers owns everything by default.
+
+# Cronet fork and Cronvoy sources + tests
+library/java/org @alyssawilk @carloseltuerto
```

Comment:
Of course! It's easier to have these types of conversations with something in front of us all.

My understanding of the `CODEOWNERS` file is that the envoy-mobile-maintainers team would still have global ownership? If that's not the case or if this isn't what envoy does, I'm happy to mirror the process and nominate someone from our side too!

Reply:
Oh I think the structure is totally great. I have two questions

1) (for @carloseltuerto) who we want the owners to be.  I'm happy to co-own but as I'm not working on android and I don't know Java I'm going to be of limited use, where someone from his team might be more helpful.  We'd talked about this offline but having a PR is motivation to get it sorted, though I'm happy to be a placeholder if we don't want to block on that :-)

2) (for @buildbreaker ) what we want the policy to be for envoy-mobile CODEOWNERS. For upstream Envoy, our policy is that extensions (which have lower support bar than core code) have at least 2 owners so they can cross-review.  They're functionally trusted as OWNERS.md owners for those files, though if neither is a maintainer they have to poke someone to submit code for them because github doesn't actually allow file based permissions.  If that works for you folks I could arguably help @carloseltuerto out by merging things approved by another cronet codeowner since I'm a repo admin.  If you'd prefer to have mobile owners always be the ones merging things I'm just as happy to have fewer PRs to juggle :-P

Reply:
At this point in time there is only me working on this. This won't stay like that, for sure. Meanwhile, for business logic, I guess @DavidSchinazi should own too.

Reply:
Gotcha. There are PR ramifications here. I can add the following for now and we can make updates as we see fit:
@buildbreaker @goaway @DavidSchinazi @carloseltuerto